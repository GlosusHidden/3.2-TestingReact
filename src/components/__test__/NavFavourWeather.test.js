import React from 'react'
import renderer from 'react-test-renderer'

import { Provider } from 'react-redux'

import NavFavourWeather from '../NavFavourWeather'

import {myStore, mockStore} from './test_data/data'

it('NavFavourWeather Component Snapshot', () => {
  const renderedValue = renderer.create(
    <Provider store={mockStore}>
      <NavFavourWeather />
    </Provider>
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
