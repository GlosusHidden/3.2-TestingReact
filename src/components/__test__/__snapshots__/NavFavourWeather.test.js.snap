// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavFavourWeather Component Snapshot 1`] = `
<div
  id="nav_favour_weather"
>
  <div
    className="text"
  >
    Favourites
  </div>
  <form
    id="add_favour_form"
    onSubmit={[Function]}
  >
    <input
      className="input_text"
      name="city_name"
      onChange={[Function]}
      placeholder="Add new city"
      required={true}
      type="text"
      value=""
    />
    <input
      alt="add"
      className="img_button"
      name="submit"
      src="img/add.png"
      type="image"
      value="Submit"
    />
  </form>
</div>
`;
